src/sqlfluff/dialects/dialect_exasol.py

```diff
@@ -929,7 +910,7 @@ class DropSchemaStatementSegment(BaseSegment):
         "SCHEMA",
         Ref("IfExistsGrammar", optional=True),
         Ref("SchemaReferenceSegment"),
-        OneOf("RESTRICT", Ref.keyword("CASCADE", optional=True), optional=True),
+        Ref("DropBehaviorGrammar", optional=True),
```

Comment:
`type` of this class needs to be updated to `type = "drop_schema_statement"`

Reply:
√